konduit-serving-api/src/main/java/ai/konduit/serving/verticles/base/BaseRoutableVerticle.java

```diff
@@ -51,7 +58,8 @@ public BaseRoutableVerticle() {
      * Start an http server the port with the value configured
      * as the httpPort key found in {@link #config()}
      */
-    protected void setupWebServer() {
+    @SneakyThrows
+    protected void setupWebServer() throws Throwable {
```

Comment:
Do we need both `@SneakyThrows` _and_ `throws Throwable`? The former should make the latter unnecessary?

Reply:
We don't, removing the `throws Throwable`

Reply:
Hmm actually I think I'll keep the latter and remove the former one